src/lib/rpcApi.js

```diff
@@ -11,6 +11,29 @@ export class RpcApi {
   }
 
   call(name, params, headers) {
+    return this.jsonPost(name, params, headers);
+  }
+
+  /**
+   * This call analyses JSON RPC response.
+   * It returns promise which resolves whether 'result' field found
+   * or reject in case 'error' field found.
+   *
+   * @returns {Promise}
+   */
+  call2(name, params, headers) {
```

Comment:
I totally agree, the question is which version of call should we use? 
- old one which just returns json and UI layer has to analyze json
- call2 which try to understand json response and reject promise if error found